pywbemtools/pywbemcli/_connection_repository.py

```diff
@@ -389,15 +494,16 @@ def _write_file(self):  # pylint: disable=no-self-use
         Write the connections file if one has been loaded.
         If the dictionary is empty, it deletes the file.
 
-        If there is an existing file it is moved to filename.bak and a new
-        current file written.
+        If there is an existing file it is moved to the same name with the
+        suffix .bak and a new current file written.
+
+        Raises:
+            ConnectionFileWriteError: If the file cannot be written.
```

Comment:
Missing `s` in exception class names: `ConnectionFile...` -> `ConnectionsFile...`